exe/3scale

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env ruby
 
-require '3scale_toolbox'
+def suppress_warnings
+  original_verbosity = $VERBOSE
+  $VERBOSE = nil
+  result = yield
+  $VERBOSE = original_verbosity
+  result
+end
+
+suppress_warnings { require '3scale_toolbox' }
```

Comment:
What are the warnings?

Reply:
For ruby 2.5, deprecation warnings from `colored` and `hashie` (dependency of swagger lib) libraries

```
3scale_toolbox/vendor/bundle/ruby/2.5.0/gems/colored-1.2/lib/colored.rb:45: warning: shadowing outer local variable - value
3scale_toolbox/vendor/bundle/ruby/2.5.0/gems/hashie-3.3.2/lib/hashie/extensions/coercion.rb:16: warning: constant ::Fixnum is deprecated
3scale_toolbox/vendor/bundle/ruby/2.5.0/gems/hashie-3.3.2/lib/hashie/extensions/coercion.rb:16: warning: constant ::Bignum is deprecated
3scale_toolbox/vendor/bundle/ruby/2.5.0/gems/hashie-3.3.2/lib/hashie/extensions/coercion.rb:17: warning: constant ::Fixnum is deprecated
3scale_toolbox/vendor/bundle/ruby/2.5.0/gems/hashie-3.3.2/lib/hashie/extensions/coercion.rb:17: warning: constant ::Bignum is deprecated
```

warnings are available running tests, not to forget about them.

Reply:
Then maybe just wrap loading colored and hashie in supress_warnings ?

Reply:
But it is interesting to uncover any happening warning, at least when testing. That's why it was implemented in `exe/3scale` entrypoint script

Reply:
Thats a good point. But once we can't do anything about the warning, we should supress it no? Even in tests it would be just distraction without any real value.

Reply:
IMO warnings in development environments are always nice. If you suppress unconditionally and tomorrow we upgrade library, potentially interesting warnings could be shadowed.

Reply:
People usually freak out when they see warnings and give them unreasonable importance.
This is happening to us in APIcast. People don't see the real error for some useless warnings in the output.

Reply:
I would expect that behavior from production environment users. Not from developers on development environments.

Reply:
And yet, people who try to contribute often get stuck on these.